src/components/FilterTopGrower.js

```diff
@@ -159,7 +170,15 @@ function FilterTopGrower(props) {
                   id="Organization"
                   placeholder="Organization"
                   value={organizationId}
-                  onChange={(e) => setOrganizationId(e.target.value)}
+                  onChange={(e) => {
```

Comment:
As well as combining the two nearly identical filters into a common component, we should aim to make the individual filters components as they are often duplicated across different pages.

Reply:
Is there anything to change here?

Reply:
I'll break up the inputs and work on the filters in a separate PR